### ISSUE TITLE
using Activities .* Endpoint seems to expect 1.0 response without sending 1.0 header

### ISSUE THREAD

**Issue by bscSCORM** [adlnet/lrs-conformance-test-suite#135]
LRSs are allowed to fall back to version 0.9, but the three using Activities .* Endpoint tests are not sending the 1.x version header, and then rejecting valid 0.9 behavior:

For example, here is "using Activities Endpoint"

```
{
                                "title": "using Activities Endpoint",
                                "name": "using Activities Endpoint",
                                "requirement": "",
                                "log": "REQUEST SUPERREQUEST\n_______________________________________\nGET /ScormEngineInterface/TCAPI/default/activities HTTP/1.1\r\nAuthorization: Basic c2Nvcm06c2Nvcm0yMDA0\r\nhost: localhost:64619\r\nConnection: close\r\n\r\nRESPONSE SUPERREQUEST\n_______________________________________\nHTTP/1.1 400 Bad Request\nserver: ASP.NET Development Server/11.0.0.0\ndate: Fri, 27 Jan 2017 22:18:26 GMT\nx-aspnet-version: 4.0.30319\nx-experience-api-version: 0.9\naccess-control-allow-origin: *\naccess-control-allow-methods: HEAD,GET,POST,PUT,DELETE\naccess-control-allow-headers: Content-Type,Content-Length,Authorization,If-Match,If-None-Match,X-Experience-API-Version\naccess-control-expose-headers: ETag,Last-Modified,Cache-Control,Content-Type,Content-Length,WWW-Authenticate,X-Experience-API-Version\ncache-control: private\ncontent-type: text/html\nconnection: Close\n\nMissing required argument: activityId\n=======================================\n",
                                "status": "failed",
                                "error": "AssertionError: expected '0.9' to match /^1\\.0\\.\\d+$/",
                                "tests": []
                            }
```

And cleaned up a bit:

``` 
FAILED: using Activities Endpoint

AssertionError: expected '0.9' to match /^1\.0\.\d+$/
REQUEST SUPERREQUEST
_______________________________________
GET /ScormEngineInterface/TCAPI/default/activities HTTP/1.1
Authorization: Basic c2Nvcm06c2Nvcm0yMDA0
host: localhost:64619
Connection: close

RESPONSE SUPERREQUEST
_______________________________________
HTTP/1.1 400 Bad Request
server: ASP.NET Development Server/11.0.0.0
date: Fri, 27 Jan 2017 21:25:36 GMT
x-aspnet-version: 4.0.30319
x-experience-api-version: 0.9
access-control-allow-origin: *
access-control-allow-methods: HEAD,GET,POST,PUT,DELETE
access-control-allow-headers: Content-Type,Content-Length,Authorization,If-Match,If-None-Match,X-Experience-API-Version
access-control-expose-headers: ETag,Last-Modified,Cache-Control,Content-Type,Content-Length,WWW-Authenticate,X-Experience-API-Version
cache-control: private
content-type: text/html
connection: Close

Missing required argument: activityId
```

**Comment by bscSCORM** [comment]
A similar thing is going on with the `agents` requests as well. "using Agents Endpoint"

For the activities requests, note that the required `activityId` parameter was left out. for the `agents` request, note that there is no `agents` resource in 0.9, it's `actors` instead.

**Comment by cr8onski** [comment]
There is a fix in development branch that hasn't made it up to master yet. 
See https://github.com/adlnet/grail-issue-tracker/issues/52 and https://github.com/adlnet/lrs-conformance-test-suite/commit/88e044a8322fb90a7a6ac076e4240c9acce69bbe
I will leave this open until it gets pushed up to master.  Thanks

**Comment by bscSCORM** [comment]
@cr8onski it looks like that fix won't apply (and in fact, it looks like I was running with that fix in place, which makes sense as I tried to grab the latest development tests when I was running).

Note specifically that what's going on here is that the 0.9 LRS is responding `400` because the request that got passed through was not valid according to 0.9.

The fix expects either `400` from 1.0 or `200` from 0.9, but it won't allow `400` from 0.9, which at least in this case is valid because as I mentioned above, the request is not a valid 0.9 request.

So, the test needs to either be careful to use valid 0.9 requests, or keep track of which are valid and which aren't, or just allow `400` from 0.9 systems in general.

**Comment by bscSCORM** [comment]
verified on f735fe8cdce875d1d56d05112062955a343258d3 (but note, f735fe8cdce875d1d56d05112062955a343258d3 is only on @cr8onski's fork so far!)
